webextensions/api/storage.json

```diff
@@ -32,7 +32,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": "≤46"
```

Comment:
@queengooborg I cannot find a sub-feature indication implementation in 46. Also we also still have "true " in places, e.g. line 117